common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -306,6 +308,25 @@ def non_editable_metadata_fields(self):
         non_editable_fields.extend([LibraryContentFields.mode, LibraryContentFields.source_library_version])
         return non_editable_fields
 
+    @lazy
+    def tools(self):
+        """
+        Grab the library tools service or raise an error.
+        """
+        return self.runtime.service(self, 'library_tools')
```

Comment:
Because the block `wants('library_tools')`, rather than `need`ing it, you should handle the case where the request for the service returns `None`.